package.json

```diff
@@ -491,12 +518,12 @@
         "test": "npm run test-compile && node ./out/src/tests/runTests.js"
     },
     "extensionDependencies": [
+        "vscode.git",
         "ms-kubernetes-tools.vscode-kubernetes-tools"
     ],
     "devDependencies": {
         "@types/chai": "^4.3.16",
-        "@types/download": "^8.0.5",
-        "@types/glob": "^8.1.0",
+        "@types/decompress": "^4.2.7",
```

Comment:
I initially added the [shelljs](https://www.npmjs.com/package/shelljs) library as a dependency to take care of invoking the `draft` binary. This resulted in transitive version conflicts between the `download` and `glob` types dependencies. Either/both of these ship with their own types now, and we're not even using the `download` package, so this is really just cleaning stuff up as we find it.

We _are_ using the `decompress` library and I was getting TypeScript errors when I didn't add the types library for that. I assume that previously the types were getting included as a dependency of another library which has been removed. I could investigate that further, but since this all relates to `@types` libraries which have no impact on the compiled code, it's probably not worth delving too deeply.